go.mod

```diff
@@ -4,14 +4,19 @@ go 1.16
 
 require (
 	github.com/apparentlymart/go-cidr v1.0.0
-	github.com/google/go-cmp v0.5.2
+	github.com/google/go-cmp v0.5.5
 	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/json-iterator/go v1.1.11 // indirect
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
 	github.com/openshift/api v0.0.0-20210416094334-c22782737ea0
 	github.com/openshift/build-machinery-go v0.0.0-20210409131504-b1828cc0cdad
 	github.com/sirupsen/logrus v1.6.0
+	golang.org/x/net v0.0.0-20210510120150-4163338589ed // indirect
 	k8s.io/api v0.21.0
 	k8s.io/apimachinery v0.21.0
 	k8s.io/client-go v0.21.0
+	k8s.io/kubectl v0.21.0
```

Comment:
At first this made me a little nervous! 
But then I remembered that go mod is smart enough to only import what we actually need from kubectl.